balancer-labs/frontend-v2
# Feature/user lock composition

# Description

Includes locked veBal in "My pool share". 

## Type of change


## How should this be tested?

Given an account with locked veBal go to this pool: 

https://feature-user-lock-compositio.balancer-d6h.pages.dev/#/ethereum/pool/0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014

and check the composition table includes the locked fiat in its calculations (see example below).

## Visual context

<img width="1742" alt="Screenshot 2023-02-07 at 19 19 39" src="https://user-images.githubusercontent.com/1316240/217332198-dedd34f3-fada-4b8a-9919-82008265d2da.png">



## Checklist:
